.github/workflows/main.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           pacman -Syu --noconfirm xcb-util-wm seatd git clang meson libinput libdrm mesa libxkbcommon wayland wayland-protocols xorg-server-xwayland scdoc
 
       - name: Fetch wlroots as a subproject
-        run: git clone https://gitlab.freedesktop.org/wlroots/wlroots.git subprojects/wlroots -b 0.16.0
+        run: git clone https://gitlab.freedesktop.org/wlroots/wlroots.git subprojects/wlroots -b 0.17.0
```

Comment:
Nit: maybe we can use "0.17" here instead so that we automatically get bugfix releases.

Reply:
It would make the build a bit less deterministic if so.

Reply:
I think that's fine, we already get non-deterministic version for our dependencies via the distro's package manager.

Reply:
Done.